includes/class-customizer.php

```diff
@@ -356,7 +356,7 @@ public static function customize_save_customizer( $wp_customize ) {
 						}
 
 						// Publish the featured item and store the new ID in $old_to_new_id.
-						$old_to_new_id[ $featured_item->ID ] = self::publish_featured_item( $featured_item, $post_parent );
+						$old_to_new_id[ $featured_item->draft_id ] = self::publish_featured_item( $featured_item, $post_parent );
```

Comment:
I'm not a 100% certain of how this works, but since we've changed the $featured_item->ID above this should be $draft_id, right @redundans?